docs/Documentation/model_introduction.md

```diff
@@ -10,22 +10,47 @@
 
-Conversion is not required for ONNX, PaddlePaddle, TensorFlow Lite and TensorFlow models (check :doc:`TensorFlow Frontend Capabilities and Limitations <openvino_docs_MO_DG_TensorFlow_Frontend>`), as OpenVINO provides C++ and Python APIs for importing them to OpenVINO Runtime directly. It provides a convenient way to quickly switch from framework-based code to OpenVINO-based code in your inference application.
+Model Conversion API, specifically, the ``mo.convert_model()`` method converts a model from original framework to ``ov.Model``. The resulting ``ov.Model`` can be inferred in the same training environment (python script or Jupiter Notebook). In addition to model files, ``convert_model`` can take OpenVINO extension objects constructed directly in Python for easier conversion of operations that are not supported in OpenVINO. The ``convert_model()`` method also has a set of parameters to :doc:`cut the model <openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model>`, :doc:`set input shapes or layout <openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model>`, :doc:`add preprocessing <openvino_docs_MO_DG_Additional_Optimization_Use_Cases>`, etc.
+
+.. image:: _static/images/model_conversion_diagram.svg
+   :alt: model conversion diagram
+
+Another option to convert a model is to use ``mo`` command-line tool. ``mo`` is a cross-platform tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
+
+The resulting model is saved to IR (Intermediate Representation) which can be read with the ``ov.read_model()`` method, then be compiled and inferred.
+
+To use it, you need a pre-trained deep learning model in one of the supported formats: TensorFlow, TensorFlow Lite, PyTorch, PaddlePaddle, or ONNX. ``mo`` converts the model to the OpenVINO Intermediate Representation format (IR), which you can infer later with :doc:`OpenVINO™ Runtime <openvino_docs_OV_UG_OV_Runtime_User_Guide>`.
+
+.. note::
+
+   ``mo`` command-line tool does not infer models.
+
+
+The figure below illustrates the typical workflow for deploying a trained deep learning model:
+
+.. image:: _static/images/BASIC_FLOW_MO_simplified.svg
+
+where IR is a pair of files describing the model:
+
+* ``.xml`` - Describes the network topology.
+* ``.bin`` - Contains the weights and biases binary data.
+
+``ov.Model`` can be serialized to IR using the ``ov.serialize()`` method. The serialized IR can be further optimized using :doc:`Post-Training Optimization tool <pot_introduction>` that applies post-training quantization methods.
```

Comment:
We just talked about IR, now we immediately switch back to `ov.Model`. This insertion above about `mo` in the middle between `mo.convert_model` that produces `ov.Model` and this part describing `serialize` looks not tightly connected with the context.

Reply:
Have rearranged it with the [a4f243d](https://github.com/openvinotoolkit/openvino/pull/17454/commits/a4f243d553e7a38ada4586623acf807e46177258 )commit.